source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -2181,16 +2190,76 @@ export class NumberInputPeer extends InputPeer<Adaptive.NumberInput> {
 }
 
 export class DateInputPeer extends InputPeer<Adaptive.DateInput> {
+    internalAddPropertySheetEntries(card: Adaptive.AdaptiveCard, includeHeader: boolean) {
+        super.internalAddPropertySheetEntries(card, includeHeader);
+
+        let defaultValue = addLabelAndInput(card, "Default value:", Adaptive.TextInput);
+        defaultValue.input.placeholder = "(not set)";
+        defaultValue.input.defaultValue = this.cardElement.defaultValue;
+        defaultValue.input.onValueChanged = () => {
+            this.cardElement.defaultValue = defaultValue.input.value;
+
+            this.changed(false);
+        }
```

Comment:
this pattern is used in lots of places -- helper method?

Reply:
I have plans to make all of this a lot better. At this time, it's just the easiest and fastest way to get it done.

Reply:
Filed #2831 to make sure it isn't lost